media-video/ffmpeg/ffmpeg-4.3.1.recipe

```diff
@@ -8,11 +8,11 @@ COPYRIGHT="2000-2003 Fabrice Bellard
 LICENSE="GNU LGPL v3"
 REVISION="2"
 SOURCE_URI="https://ffmpeg.org/releases/ffmpeg-$portVersion.tar.xz"
-CHECKSUM_SHA256="1d0ad06484f44bcb97eba5e93c40bcb893890f9f64aeb43e46cd9bb4cbd6795d"
+CHECKSUM_SHA256="ad009240d46e307b4e03a213a0f49c11b650e445b1f8be0dda2a9212b34d2ffb"
 PATCHES="ffmpeg-$portVersion.patchset"
 ADDITIONAL_FILES="gcc_runtime.c"
 
-ARCHITECTURES="!x86_gcc2 x86 !x86_64"
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
 SECONDARY_ARCHITECTURES="x86_gcc2 !x86"
```

Comment:
How can x86_gcc2 be enabled for secondary architecture if it's disabled for primary architecture?
Has this been tested on 32 bit also?